source/localizable/components/triggering-changes-with-actions.md

```diff
@@ -426,3 +403,22 @@ export default Ember.Component.extend({
   }
 });
 ```
+Then our `system-preferences-editor` template passes its local `deleteUser` action into the `user-profile` as that
+component's `deleteCurrentUser` property.
+
+```app/templates/components/system-preferences-editor.hbs
+{{user-profile deleteCurrentUser=(action 'deleteUser' login.currentUser.id)}}
+```
+The action `deleteUser` is in quotes, since `system-preferences-editor` is where the action is defined now. Quotes simply indicate that the action should be looked for in that comoponent's `actions:{}` object, rather than in the `attrs` that have been passed from the parent.
```

Comment:
See https://github.com/emberjs/guides/blob/master/CONTRIBUTING.md for guidelines on how to break up the text.